### PR TITLE
Display plugin content on repository overview page

### PR DESCRIPTION
**Pull request by Pilopa**
## Proposed changes

Usage analysis has revealed that the mySCM page should be integrated into the repository overview. This requires a more compact layout that is adjusted to match the repositories page visually. Additionally, users should be able to hide individual widgets through a configuration page.

### Your checklist for this pull request

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

**Contributor**:
- [x] PR is well described and the description can be used as a commit message on squash
- [x] Related issues linked to PR if existing and labels set
- [x] New code is covered with unit tests
- [ ] New ui components are tested inside the storybook (module ui-components only) 
- [x] [Changelog entry file](https://github.com/scm-manager/changelog#changelog-entry-files) created in `gradle/changelog` or CHANGELOG.md is updated for plugins
- [ ] Feature has been tested with different permissions
- [ ] Documentation updated (only necessary for new features or changed behaviour)

**Reviewer**:
- [ ] The clean code principles are respected ([CleanCode](https://clean-code-developer.com/virtues/))
- [ ] All new code/logic is implemented on the right spot / "Should this be done here?"
- [ ] UI changes fits into the layout
- [ ] The UI views / components are responsive (mobile views)
- [ ] Correct translations are available

### Checklist for branch merge request (not required for forks)

- [ ] Branch is green/blue on [Jenkins](https://oss.cloudogu.com/jenkins/)
- [ ] Quality Gate passed on [SonarQube](https://sonarcloud.io/organizations/scm-manager/projects)

### CODE REVIEW

File: src/main/js/favoriteRepositories/MyFavoriteRepositories.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  }
+
+  if (!data || !data.repositories || data.repositories.length === 0) {
+    return (
+      <>
+        {header}
+        <Notification type="info">{t("scm-landingpage-plugin.favoriteRepository.noData")}</Notification>
+      </>
+    );
+  }
```
Comment (eheimbuch) [inline src/main/js/favoriteRepositories/MyFavoriteRepositories.tsx:72]: This should be refactored. No need for 3 If cases.

File: src/main/js/CollapsibleContainer.tsx [hunk 2]
```diff
@@ -66,17 +77,27 @@ const CollapsibleContainer: FC<Props> = ({ title, separatedEntries, emptyMessage
...
+    onCollapseToggle && onCollapseToggle();
+  };
+
   return (
     <Container>
-      <div className="has-cursor-pointer" onClick={() => setCollapsed(!collapsed)}>
+      <div className="has-cursor-pointer" onClick={onClick}>
         <Headline>
-          <Icon name={icon} color="default" /> {title}
+          <Icon name={icon} color="default" /> {title} <Tag color="info">{count || 0}</Tag>
```
Comment (eheimbuch) [inline src/main/js/CollapsibleContainer.tsx:100]: I'm not satisfied with the tag yet. We should make the font bold and maybe a little bigger?

**Review by eheimbuch — CHANGES_REQUESTED** [review]

**Comment by eheimbuch** [comment]
Some more findings:
- Make the config page icon one size larger and fix the positioning
- Make the font in the count tag bold and fix the tag positioning
- On the mobile view collapse all categories per default if no local storage key exists yet
- Fix the text overflow in the pull request card on 200% zoom
- Find a better solution for the DateFromNow Component as can break the card layout if it doesn't get enough space

**Comment by Pilopa** [comment]
- [x] >     * Fix the text overflow in the pull request card on 200% zoom

**Pull request merged**